inlong-agent/agent-common/pom.xml

```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>org.apache.inlong</groupId>
         <artifactId>inlong-agent</artifactId>
-        <version>1.3.0-SNAPSHOT</version>
```

Comment:
the version changing in pom file could use another PR.

Reply:
see https://github.com/apache/inlong/pull/5656